microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

```diff
@@ -13,7 +13,13 @@ ms.date: 02/17/2021
 <!-- ====================================================================== -->
 ## Exhibit G: Terms and Conditions for Microsoft Edge Extensions
 
-These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum") is a supplement to the terms and conditions of the App Developer Agreement, the current version of which can be found at [https://go.microsoft.com/fwlink/p/?LinkID=221922](developer-policies.md).  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
+These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum")  is a supplement to the terms and conditions of the Windows [App Developer Agreement](https://go.microsoft.com/fwlink/p/?LinkID=221922) together with the parallel [Microsoft Edge Add-ons store developer policies](developer-policies.md).
+
+*  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.
+
```

Comment:
@Reezaali  @mikehoffms  - The link change to ADA seems like a bug that and fixing that should not require a CELA approval. Are there any other major changes to this document apart from the link to ADA? If not, I think this does not require CELA approval, we should go ahead.

Reply:
The present 2101 is superseded by a couple newer PRs, see equivalent comment from Ashish in new PR
https://github.com/MicrosoftDocs/edge-developer/pull/2273#discussion_r1007961529

In the new PR that affects this article, there are no changes to this article other than this link fix to make the link actually go to the page that the link text claims that the link goes to.